engine/runtime/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -238,7 +238,7 @@ object ProgramExecutionSupport {
     val message = error match {
       case _: ThreadInterruptedException =>
         val message = s"Execution of function $itemName interrupted."
-        ctx.executionService.getLogger.log(Level.WARNING, message)
+        ctx.executionService.getLogger.log(Level.FINE, message)
```

Comment:
This is `java.util.logging.Logger`, right? Then the message could be ``"Execution of function {0} interrupted."` and the `itemName` could be sent to the logger as parameter. This applies to other Java logged messages.